gluten-core/src/main/scala/io/glutenproject/expression/UnaryExpressionTransformer.scala

```diff
@@ -215,3 +215,27 @@ case class RandTransformer(
     ExpressionBuilder.makeScalarFunction(functionId, inputNodes, typeNode)
   }
 }
+
+case class GetArrayStructFieldsTransformer(
+    substraitExprName: String,
+    child: ExpressionTransformer,
+    ordinal: Int,
+    numFields: Int,
+    containsNull: Boolean,
```

Comment:
It seems numFields and containsNull are not added to the function arguments. Are they needed?

Reply:
numFields and containsNull  are not needed in CH backend.